policies/resource_cluster_policy.go

```diff
@@ -125,6 +125,9 @@ func ResourceClusterPolicy() *schema.Resource {
 			"max_clusters_per_user": {
 				Type:     schema.TypeInt,
 				Optional: true,
+				Description: "Max number of clusters per user that can be active\n" +
+				    "using this policy. If not present, there is no max limit.",
+				ValidateFunc: validation.IntAtLeast(1),
```

Comment:
does it work when omitted?

Reply:
Yes, I've tested it with and without the parameter.